soh/src/overlays/misc/ovl_kaleido_scope/z_kaleido_item.c

```diff
@@ -764,8 +779,8 @@ void KaleidoScope_DrawItemSelect(PlayState* play) {
     gDPSetCombineLERP(POLY_KAL_DISP++, PRIMITIVE, ENVIRONMENT, TEXEL0, ENVIRONMENT, TEXEL0, 0, PRIMITIVE, 0, PRIMITIVE,
                       ENVIRONMENT, TEXEL0, ENVIRONMENT, TEXEL0, 0, PRIMITIVE, 0);
 
-    for (i = 0; i < 15; i++) {
-        if ((gAmmoItems[i] != ITEM_NONE) && (gSaveContext.inventory.items[i] != ITEM_NONE)) {
+    for (i = 0; i < (CVarGetInteger("gBetterAmmoRendering", 0) ? 24 : 15); i++) {
+        if ((CVarGetInteger("gBetterAmmoRendering", 0) ? ItemInSlotUsesAmmo(i) : gAmmoItems[i] != ITEM_NONE) && (gSaveContext.inventory.items[i] != ITEM_NONE)) {
```

Comment:
You can move the CVar get to a local var outside the for loop so we aren't incurring the cost of two CVar calls per loop execution.

```suggestion
    u8 gBetterAmmoRendering = CVarGetInteger("gBetterAmmoRendering", 0);

    for (i = 0; i < (gBetterAmmoRendering ? 24 : 15); i++) {
        if ((gBetterAmmoRendering ? ItemInSlotUsesAmmo(i) : gAmmoItems[i] != ITEM_NONE) && (gSaveContext.inventory.items[i] != ITEM_NONE)) {
```